src/system/time/services/rtc.cpp

```diff
@@ -7,8 +10,21 @@
  * @return false On failure
  */
 bool TimeProviderRTC::init() {
-    this->datetime = { 0, 0, 0, 0, 0, 0 };
-    return true;
+    if (! rtc.begin()) {
+        Serial.println("Couldn't find RTC");
+        Serial.flush();
+        return false;
+    }
+    else{
```

Comment:
This else is not necesary, since you are already returning if the rtc is not beginned.